applications/serial_lte_modem/doc/FTP_AT_commands.rst

```diff
@@ -110,3 +110,108 @@ Test command
 ------------
 
 The test command is not supported.
+
+TFTP client #XTFTP
+==================
+
+The ``#XTFTP`` command allows you to send TFTP commands.
+
+   .. note::
+      The maximum size of a file supported is 2100 bytes.
+      TFTP write request is not supported yet.
+
+Set command
+-----------
+
+The set command allows you to send TFTP commands based on RFC1350.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XTFTP=<op>,<url>,<port>,<file_path>[,<mode>]
+
+* The ``<op>`` parameter can accept one of the following values:
+
+  * ``1`` - TFTP read request using IP protocol family version 4.
+  * ``2`` - TFTP write request using IP protocol family version 4 (currently not supported).
+  * ``3`` - TFTP read request using IP protocol family version 6.
+  * ``4`` - TFTP write request using IP protocol family version 6 (currently not supported).
```

Comment:
```suggestion
  * ``1`` - TFTP read request using IP protocol family version 4.
  * ``3`` - TFTP read request using IP protocol family version 6.
  * *Currently not supported:* ``2`` - TFTP write request using IP protocol family version 4.
  * *Currently not supported:* ``4`` - TFTP write request using IP protocol family version 6.
```
I'm not sure if we even need to include the values that are currently not supported. But if they are included, it's better to start with the "not supported" text so it stands out more.